docs/Metrics/DeployTime.md

```diff
@@ -16,11 +16,20 @@ The time it takes from when a PR is merged to when it is deployed.
 
 
 ## How is it calculated?
-You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
-
+1. You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+2. Connect pr/mr to the right `deployment`
+![](/img/Metrics/pr-commit-deployment-relation.jpeg)
+   1. Use plugin `refdiff` to calculate `refs_commits_diffs` between any two consecutive deployments under the same scope,
```

Comment:
We don't need to emphasize 'Use plugin `refdiff` to calculate `refs_commits_diffs`' as it's a user manual, just tell how it is calculated is OK.

For example:
Find the [commits diff](the link to refdiff plugin) between two consecutive deployments by deployments' commit_sha.